src/platform/elements/chips/Chips.ts

```diff
@@ -66,7 +80,7 @@ export class NovoChipElement {
 
 @Component({
   selector: 'chips,novo-chips',
-  providers: [CHIPS_VALUE_ACCESSOR],
+  providers: [CHIPS_VALUE_ACCESSOR, ChipsStateService],
```

Comment:
Would it make sense to call this InputStateService? so that maybe it could be reusable?

Reply:
yes. i was actually thinking similarly too.

Reply:
and done